### PR TITLE
CMQ-2300: Move Directions link

### PR DESCRIPTION
**Pull request by czeczotka**

### CODE REVIEW

**Review by edgarjimenez — APPROVED** [review]

File: source/change-log/index.html.md.erb [hunk 1]
```diff
@@ -13,22 +13,22 @@ description: Find out what has changed in the specification and make sure your a
 
  * Clarification on format for Gov-Client-Timezone
 
-This will be updated in the <a href="https://www.gov.uk/government/publications/direction-under-regulation-22-of-the-delivery-of-tax-information-through-software-ancillary-metadata-regulations-2019-si-2019360">Directions</a>.
+This will be updated in the Directions. To check what you need to send, <a href="../connection-method/" onclick="ga('send', 'event', 'On Page Navigation', 'Change Log', 'Select your connection method');">select your connection method</a>.
```
Comment (matthewmascord) [inline source/change-log/index.html.md.erb:16]: I could be wrong but this ga function is deprecated and no longer supported. It went away when tracking consent was rolled out.

File: source/change-log/index.html.md.erb [hunk 2]
```diff
@@ -13,22 +13,22 @@ description: Find out what has changed in the specification and make sure your a
...
 
 #### 31 January 2022
 
  * For the web application via server connection method, you do not need to submit values for Gov-Client-Browser-Plugins, Gov-Client-Local-IPs or Gov-Client-Local-IPs-Timestamp. If your application submits values for these headers it is not an issue.
  * For all connection methods other than web application via server, you still need to submit values for Gov-Client-Local-IPs and Gov-Client-Local-IPs-Timestamp
 
-This will be updated in the <a href="https://www.gov.uk/government/publications/direction-under-regulation-22-of-the-delivery-of-tax-information-through-software-ancillary-metadata-regulations-2019-si-2019360">Directions</a>.
+This will be updated in the Directions. To check what you need to send, <a href="../connection-method/" onclick="ga('send', 'event', 'On Page Navigation', 'Change Log', 'Select your connection method');">select your connection method</a>.
```
Comment (matthewmascord) [inline source/change-log/index.html.md.erb:23]: See comment above.

File: source/change-log/index.html.md.erb [hunk 3]
```diff
@@ -38,7 +38,7 @@ This will be updated in the <a href="https://www.gov.uk/government/publications/
 
 * Clarification on the format for Gov-Client-Public-IP-Timestamp and Gov-Client-Local-IPs-Timestamp
 
-This will be updated in the <a href="https://www.gov.uk/government/publications/direction-under-regulation-22-of-the-delivery-of-tax-information-through-software-ancillary-metadata-regulations-2019-si-2019360">Directions</a>.
+This will be updated in the Directions. To check what you need to send, <a href="../connection-method/" onclick="ga('send', 'event', 'On Page Navigation', 'Change Log', 'Select your connection method');">select your connection method</a>.
```
Comment (matthewmascord) [inline source/change-log/index.html.md.erb:41]: See comment above.

File: source/change-log/index.html.md.erb (hunk 1 above)
Comment (czeczotka) [reply source/change-log/index.html.md.erb:16]: We use it 26 times in the repo so should discuss separately. For now I put the same link to `select connection method` as in other places.

**Review by matthewmascord — APPROVED** [review]

**Pull request merged**